src/utils/workflow-utils.ts

```diff
@@ -4,32 +4,32 @@ import { CallSupabaseResponse } from 'services/supabase';
 import { ResourceConfigDictionary } from 'stores/ResourceConfig/types';
 import { Schema } from 'types';
 
-const mergeResourceConfigs = (
-    queryData: DraftSpecQuery,
-    resourceConfig: ResourceConfigDictionary,
-    restrictedDiscoveredCollections: string[]
-): ResourceConfigDictionary => {
-    const existingCollections = Object.keys(resourceConfig);
-    const mergedResourceConfig: ResourceConfigDictionary = {};
+// const mergeResourceConfigs = (
```

Comment:
Given the temperature of our conversations around the _future of discovery_, I was unsure if this helper function should be commented out or removed entirely. It is in the commit record, so it can be referenced there if it becomes relevant in the future. Consequently, I would vote to delete this commented out block. Is that alright with you?